protobuf_benchmarks/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+name: protobuf_benchmarks
+description: Benchmarks various protobuf functions
+publish_to: none
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  protobuf: any
+
+dev_dependencies:
+  benchmark_harness: ^2.0.0
+  js: ^0.6.3
+  lints: ^1.0.0
+  protoc_plugin:
+    path: '../protoc_plugin'
```

Comment:
Could you align this with `query_benchmark/pubspec.yaml` (it has e.g. a `dependency_override` for `package:protobuf`).

It may be sufficient to make `protobuf` also a `dev_dependency` with a `path: ` since we don't publish these benchmark packages.

Reply:
> It may be sufficient to make protobuf also a dev_dependency with a path: since we don't publish these benchmark packages.

That doesn't work:

```
Because protobuf_benchmarks depends on protoc_plugin from path which depends on protobuf ^2.0.0, protobuf from hosted is required.
So, because protobuf_benchmarks depends on protobuf from path, version solving failed.
```

Reply:
I added a dep override as in query_benchmark.